option optimize_for = LITE_RUNTIME;

message ShopPlayBuyRequest
{
 required uint32 id = 1;
 required uint32 num = 2;
}
